infrastructure/development/README.md

```diff
@@ -0,0 +1,27 @@
+# Infrastructure
+
+## Guidelines
```

Comment:
I would keep these gotchas at a higher level since they are not specific for the `development` environment

Reply:
moved up in commit 92205aeaa173d6b6757507164e517950c01c1afc